vcf-dumper/vcf-dumper-ws/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<web-app version="3.0" xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd">
+
+    <display-name>European Variation Archive VCF Dumper</display-name>
+
+    <servlet>
+        <servlet-name>VCFDumperServer</servlet-name>
+        <servlet-class>org.springframework.web.servlet.DispatcherServlet</servlet-class>
+        <init-param>
+            <param-name>contextClass</param-name>
+            <param-value>org.springframework.web.context.support.AnnotationConfigWebApplicationContext</param-value>
+        </init-param>
+        <load-on-startup>1</load-on-startup>
+    </servlet>
+
```

Comment:
Do we really need to still use a web.xml when we are already using latest version of Spring-boot and servlet 3.0?

This could lead to have some configuration parameters through code or annotations and others through web.xml